examples/16-3-release-blog-post/create-ref-example.js

```diff
@@ -0,0 +1,14 @@
+class MyComponent extends React.Component {
+  // highlight-next-line
+  divRef = React.createRef();
```

Comment:
Maybe this should be called `inputRef`? PR: https://github.com/bvaughn/reactjs.org/pull/2

Reply:
D'oh